rfcs/0000-nix-mark-stale-issues.md

```diff
@@ -0,0 +1,109 @@
+---
+feature: nix-mark-stale-issues
+start-date: 2022-04-18
+author: John Ericson (@Ericson2314)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Do not auto-close Nix issues and PRs.
+
+Make the policy clear, and harmonize Nix's with Nixpkgs's, which was approved by
+[RFC 51](https://github.com/NixOS/rfcs/blob/master/rfcs/0051-mark-stale-issues.md).
+
+# Motivation
+[motivation]: #motivation
+
+## Background
+
+In 2019, after much discussion,
+[RFC 51](https://github.com/NixOS/rfcs/blob/master/rfcs/0051-mark-stale-issues.md) was approved to mark old Nixpkgs issues and PRs as stale,
+but not close anything.
+
+In 2021, @garbas contributed [Nix PR 4526](https://github.com/NixOS/nix/pull/4526),
+which initially just made a stale bot configuration in line with Nixpkgs.
+But @edolstra [specifically requested](https://github.com/NixOS/nix/pull/4526#discussion_r571687438) that stale items do be closed.
+To be clear, there *is* no formal violation, as that RFC was just about Nixpkgs, but we should reconsider whether this was a good idea.
+
+## Clear policy
+
+Firstly, whatever the policy is, it should be clearly announced.
+
+Without a big announcement of this decision, many of us were caught of guard when issues were closed the other day.
+Anyone that just saw the previous stale bot "marking stale" messages probably assumed stale bot was configured like Nixpkgs.
```

Comment:
Especially true given that the message was not adapted to differ from nixpkgs and explain what would happen!